src/task_service.ts

```diff
@@ -0,0 +1,134 @@
+import {IIdentity} from '@essential-projects/iam_contracts';
+import {
+  FlowNodeInstance,
+  FlowNodeInstanceState,
+  IFlowNodeInstanceService,
+} from '@process-engine/flow_node_instance.contracts';
+import {APIs, DataModels} from '@process-engine/management_api_contracts';
+
+import {EmptyActivityConverter, ManualTaskConverter, UserTaskConverter} from './converters/index';
+
+export class TaskService implements APIs.ITaskManagementApi {
+
+  private readonly flowNodeInstanceService: IFlowNodeInstanceService;
+
+  private readonly userTaskConverter: UserTaskConverter;
+  private readonly manualTaskConverter: ManualTaskConverter;
+  private readonly emptyActivityConverter: EmptyActivityConverter;
+
+  constructor(
+    flowNodeInstanceService: IFlowNodeInstanceService,
+    emptyActivityConverter: EmptyActivityConverter,
+    manualTaskConverter: ManualTaskConverter,
+    userTaskConverter: UserTaskConverter,
+  ) {
+    this.flowNodeInstanceService = flowNodeInstanceService;
+
+    this.userTaskConverter = userTaskConverter;
+    this.manualTaskConverter = manualTaskConverter;
+    this.emptyActivityConverter = emptyActivityConverter;
+  }
+
+  public async getAllSuspendedTasks(identity: IIdentity): Promise<DataModels.Tasks.TaskList> {
+
+    const suspendedFlowNodes = await this.flowNodeInstanceService.queryByState(FlowNodeInstanceState.suspended);
+
+    const userTaskList = await this.userTaskConverter.convert(identity, suspendedFlowNodes);
+    const manualTaskList = await this.manualTaskConverter.convert(identity, suspendedFlowNodes);
+    const emptyActivityList = await this.emptyActivityConverter.convert(identity, suspendedFlowNodes);
+
+    const taskList: DataModels.Tasks.TaskList = {
+      emptyActivities: emptyActivityList.emptyActivities,
+      userTasks: userTaskList.userTasks,
+      manualTasks: manualTaskList.manualTasks,
+    };
+
+    return taskList;
+  }
+
+  public async getTasksForProcessModel(identity: IIdentity, processModelId: string): Promise<DataModels.Tasks.TaskList> {
```

Comment:
Wenn die Änderungen in den Contracts durch sind:

```suggestion
  public async getSuspendedTasksForProcessModel(identity: IIdentity, processModelId: string): Promise<DataModels.FlowNodeInstances.TaskList> {
```